Robust.Server/ServerStatus/WatchdogApi.cs

```diff
@@ -80,6 +80,9 @@ private async Task<bool> UpdateHandler(IStatusHandlerContext context)
             return true;
         }
 
+        /// <remarks>
+        /// This function is used by https://github.com/tgstation/tgstation-server. Notify the project maintainer(s) if this API is changed.
```

Comment:
GitHub is stupid and will append the period to the link, making it invalid if you click it

```suggestion
        /// This function is used by https://github.com/tgstation/tgstation-server
        /// Notify the project maintainer(s) if this API is changed.
```